docs/index.rst

```diff
@@ -3,28 +3,20 @@ Welcome to Remix's documentation!
 
 **Remix IDE** is used for the entire journey of smart contract development by users at every knowledge level. 
 It requires no setup, fosters a fast development cycle, and has a rich set of plugins with intuitive GUIs.  
-The IDE comes in two flavors (web app or desktop app) and as a VSCode extension.
+The IDE comes in two flavors (web app or desktop app).
 
 **Remix Online IDE**, see: `https://remix.ethereum.org <https://remix.ethereum.org>`__
 
 Supported browsers: Firefox, Chrome, Brave. We do not support use of Remix on tablets or mobile devices.
 
-**Remix Desktop IDE**, see releases: `https://github.com/ethereum/remix-desktop/releases <https://github.com/ethereum/remix-desktop/releases>`__
-
-**Ethereum Remix**, the VSCode extension, see `here <https://marketplace.visualstudio.com/items?itemName=RemixProject.ethereum-remix>`__.  
-Documentation for the VSCode extension is located `here <https://github.com/ethereum/remix-vscode#ethereum-remix-project-extension-for-visual-studio-code>`__.
-
-**Remix Documentation Translations** The Remix docs are currently in `English <https://remix-ide.readthedocs.io/en/latest/>`__ and `Simplified Chinese <https://remix-ide.readthedocs.io/zh_CN/latest/>`__.  More languages are on the way.
+**Translations of the Remix Documentation ** currently in `English <https://remix-ide.readthedocs.io/en/latest/>`__, `Simplified Chinese <https://remix-ide.readthedocs.io/zh_CN/latest/>`__, `Spanish <https://remix-ide.readthedocs.io/es/latest/>`__, `Italian <https://remix-ide.readthedocs.io/it/latest/>`__, and `French <https://remix-ide.readthedocs.io/fr/latest/>`__.  More languages are on the way.
```

Comment:
Do you want to say "Translations of the Remix Documentation are currently available in..."

Reply:
For this page, I'm going in the shorthand style.